.github/workflows/restart.yml

```diff
@@ -7,34 +7,39 @@ on:
     - cron: '7/15 * * * *'
 
 jobs:
-  # restart-staging:
-  #   concurrency: staging
-  #   name: restart (staging)
-  #   environment: staging
-  #   runs-on: ubuntu-latest
-  #   env:
-  #     APP_URL: https://inventory-stage-datagov.app.cloud.gov
-  #   steps:
-  #     - name: checkout
-  #       uses: actions/checkout@v2
-  #     - name: restart
-  #       uses: cloud-gov/cg-cli-tools@main
-  #       with:
-  #         command: cf restart inventory --strategy rolling
-  #         cf_org: gsa-datagov
-  #         cf_space: staging
-  #         cf_username: ${{secrets.CF_SERVICE_USER}}
-  #         cf_password: ${{secrets.CF_SERVICE_AUTH}}
-  #     - name: smoke test
-  #       run: bin/smoke.sh
-  #     - name: restart proxy
-  #       uses: cloud-gov/cg-cli-tools@main
-  #       with:
-  #         command: cf restart inventory-proxy --strategy rolling
-  #         cf_org: gsa-datagov
-  #         cf_space: staging
-  #         cf_username: ${{secrets.CF_SERVICE_USER}}
-  #         cf_password: ${{secrets.CF_SERVICE_AUTH}}
+  restart-staging:
+    concurrency: staging
+    name: restart (staging)
+    environment: staging
+    runs-on: ubuntu-latest
+    env:
+      APP_URL: https://inventory-stage-datagov.app.cloud.gov
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: restart
+        uses: cloud-gov/cg-cli-tools@main
+        with:
+          command: cf restart inventory --strategy rolling
+          cf_org: gsa-datagov
+          cf_space: staging
+          cf_username: ${{secrets.CF_SERVICE_USER}}
+          cf_password: ${{secrets.CF_SERVICE_AUTH}}
+      - name: smoke test
+        run: bin/smoke.sh
+      - name: restart proxy
+        uses: cloud-gov/cg-cli-tools@main
+        with:
+          command: >
+            cf scale inventory -i 1 &&
+            sleep 5 &&
+            cf restart inventory-proxy --strategy rolling' &&
+            sleep 5;
+            cf scale inventory -i 2
```

Comment:
Is the `>` really necessary?  Github actions hasn't complained about multi-line commands before. Usually I see people put a `|` instead of the `>`.  I guess it doesn't matter.  I was just curious.

Reply:
have not tried without `>`, but the difference in `>, |, >- |-` is all about line breaks, which does not matter in this case.